Update/kage_003.txt

```diff
@@ -229,43 +229,43 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500237", 256, TRUE);
 	OutputLine(NULL, "「それに、自分の職務は犯罪事件を捜査することであって、犯罪自体の存在の有無を確かめることではありません。",
-			NULL, "\"Besides, my job is to investigate crimes, not confirm the existence of a crime itself.", Line_WaitForInput);
+			NULL, "\"Besides, as part of the criminal investigations unit, my job's to investigate criminal offenses, not figure out whether a crime happened in the first place.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500238", 256, TRUE);
 	OutputLine(NULL, "その点は、どうかご理解ください」",
-			NULL, " So, please understand that point.\"", Line_Normal);
+			NULL, " So, please keep that in mind.\"", Line_Normal);
 	ClearMessage();
 
 //赤坂rvS08/13/321000308.「…失礼しました。kvS08/13/321000309.つい、気持ちが急いてしまって」
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 13, "ps3/s08/13/321000308", 256, TRUE);
 	OutputLine(NULL, "「…失礼しました。",
-			NULL, "\"...I'm terribly sorry.", Line_WaitForInput);
+			NULL, "\"...I'm sorry for bringing this up then.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 13, "ps3/s08/13/321000309", 256, TRUE);
 	OutputLine(NULL, "つい、気持ちが急いてしまって」",
-			NULL, " I let my emotions run out of control.\"", Line_Normal);
+			NULL, " Seems like I let my feelings get the best of me here.\"", Line_Normal);
 	ClearMessage();
 
 //r少々、熱くなりすぎて話が脱線したと、私は咳払いして気を落ち着ける。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "少々、熱くなりすぎて話が脱線したと、私は咳払いして気を落ち着ける。",
-			NULL, "It started to get hot as the conversation derailed, so I cleared my throat to calm myself down.", Line_Normal);
+			NULL, "Before I knew it, the conversation trailed off and the temperature started getting unreasonably warm, so I cleared my throat while trying to pull myself together.", Line_Normal);
 	ClearMessage();
 
 //r南井警部の見解は保守的ではなく、もっともなことだ。この地域の刑事課に他県のガス災害、そして陰謀云々を語ったところで、職務としてできることは何もない。
 	if (GetGlobalFlag(GADVMode)) {
 		OutputLineAll("", NULL, Line_ContinueAfterTyping);
 		OutputLine(NULL, "南井警部の見解は保守的ではなく、もっともなことだ。この地域の刑事課に他県のガス災害、そして陰謀云々を語ったところで、職務としてできることは何もない。",
-				NULL, "<size=-2>Chief Inspector Minai wasn't being conservative, her views were absolutely valid. Even though I informed the criminal investigators in this area about the conspiracies related to the gas disaster, it's not their duty to act on that information.", Line_Normal);
+				NULL, "<size=-2>Inspector Minai wasn't purposefully being dismissive, her opinions are completely valid. Just because I discuss the possiblity of a conspiracy surrounding another town's gas disaster, doesn't mean that they have to follow up on it, especially when they don't even work in that prefecture.", Line_Normal);
```

Comment:
```suggestion
				NULL, "<size=-2>Chief Inspector Minai wasn't purposefully being dismissive, her opinions are completely valid. Just because I discuss the possibility of a conspiracy surrounding another town's gas disaster, doesn't mean that they have to follow up on it, especially when they don't even work in that prefecture.", Line_Normal);
```